tests/src/ut_test_kvp.c

```diff
@@ -342,6 +364,18 @@ static int test_ut_kvp_freeGlobalInstance( void )
     return 0;
 }
 
+static int test_ut_kvp_getPlatformProfileInstance( void )
```

Comment:
u don't need this anymore right? Since you never need to load the profile as part of Suite 6 do you?